src/components/radio-group/radio-group.ts

```diff
@@ -0,0 +1,91 @@
+import { html, LitElement } from 'lit';
+import { property } from 'lit/decorators.js';
+import { IgcRadioComponent } from '../radio/radio';
+import { styles } from './radio-group.css';
+import { watch } from '../common/decorators';
+
+export class IgcRadioGroupComponent extends LitElement {
+  static styles = styles;
+
+  private _cachedRadios!: IgcRadioComponent[];
+
+  private get _slottedRadios() {
+    const slot = this.shadowRoot!.querySelector('slot');
+    const childNodes = slot!.assignedNodes({ flatten: true });
+    return Array.prototype.filter.call(childNodes, (node) => {
+      return node.nodeName.toLowerCase() == 'igc-radio';
+    }) as IgcRadioComponent[];
+  }
```

Comment:
```suggestion
@queryAssignedNodes(undefined, true, 'igc-radio')
  _slottedRadios!: NodeListOf<IgcRadioComponent>;
```

I think this should also do the trick